WordPress/Classes/ViewRelated/Stats/Shared Views/GrowAudienceCell.swift

```diff
@@ -143,18 +143,29 @@ class GrowAudienceCell: UITableViewCell, NibLoadable {
 
     private enum Strings {
 
-        static let viewsCountDescriptionSingular =
-            NSLocalizedString("View to your site so far", comment: "Description for view count. Singular.")
-
-        static let viewsCountDescriptionPlural =
-            NSLocalizedString("Views to your site so far", comment: "Description for view count. Singular.")
-
-        static let tipTitle =
-            NSLocalizedString("A tip to grow your audience",
-                              comment: "A hint to users about growing the audience for their site, when their site doesn't have many views yet.")
-
-        static let dismissButtonTitle =
-            NSLocalizedString("Dismiss", comment: "Title for button that will dismiss the Grow Your Audience card.")
+        static let viewsCountDescriptionSingular = NSLocalizedString(
+            "growAudienceCell.viewsCount.singular",
+            value: "View to your site so far",
+            comment: "Description for view count. Singular."
+        )
+
+        static let viewsCountDescriptionPlural = NSLocalizedString(
+            "growAudienceCell.viewsCount.plural",
+            value: "Views to your site so far",
+            comment: "Description for view count. Singular."
+        )
```

Comment:
Unrelated to this PR, but it might be worth to explore [using .stirngsdict to localize plurals](https://developer.apple.com/documentation/xcode/localizing-strings-that-contain-plurals).